### PR TITLE
added data structure questions

### PR DESCRIPTION
**Pull request by Shrutikabansal**
### **Describe your change:**

Added Algorithm related to DynamicPrograming and Backtracking

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: data_structures/BitManipulation/Different_Bits_Sum_Pairwise.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+Example Input
+Input 1:
+A = [1, 3, 5]
+       
+
+
+Example Output
+Ouptut 1:
```
Comment (cclauss) [inline data_structures/BitManipulation/Different_Bits_Sum_Pairwise.py:28]:
```suggestion
Output 1:
```
https://github.com/TheAlgorithms/Python/pull/1845/checks?check_run_id=573851978#step:5:10

File: data_structures/Math/PowerfulArray.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+0
+   
+
+
+Example Explanation
+Explanation 1:
+Power of A = 24 and Power of B = 48.
+So, the answer is 0.
+'''
+
```
Comment (cclauss) [inline data_structures/Math/PowerfulArray.py:39]:
```suggestion

import math
```
https://travis-ci.com/github/TheAlgorithms/Python/builds/159594158#L339

**Comment by cclauss** [comment]
data_structures/Backtracking/sudoku.cpp does not belong in this repoo.  It belongs in https://github.com/TheAlgorithms/C-Plus-Plus

**Comment by cclauss** [comment]
https://github.com/TheAlgorithms/Python/blob/master/Travis_CI_tests_are_failing.md

**Comment by cclauss** [comment]
___This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.___

**Comment by Shrutikabansal** [comment]
"This PR should only change one algorithm file. To ease review, please open separate PRs for separate algorithms." I got this point but why checks are failing

**Comment by cclauss** [comment]
> Why checks are failing?
https://github.com/TheAlgorithms/Python/blob/master/Travis_CI_tests_are_failing.md

**Comment by Shrutikabansal** [comment]
Okay I'll do again and then raise the pr
Thank you so much

**Comment by cclauss** [comment]
There is a standalone Python file in the CPython distribution called reindent.py that can autofix those pesky tabs... https://github.com/python/cpython/blob/master/Tools/scripts/reindent.py

**Pull request closed without merging**